setup/options.rst

```diff
@@ -0,0 +1,356 @@
+
+  .. image:: _static/custom_currency.png
+
+* **Distance and Speed**
+
+  Used to select the units for distances and speed. Can be one of three values:
+
+  * *Imperial* - ``ft`` (feet) and ``mph`` (miles per hour)
+  * *Metric* - ``m`` (metres) and ``km/h`` (kilometres per hour)
+  * *SI* - ``m`` (metres) and ``m/s`` (metres per second)
+
+* **Temperature**
+
+  Used to select the units for temperature. Can be either ``Celsius`` or ``Fahrenheit``.
+
+* **Height Labels**
+
+  Used to select whether to show unit values (``+1``, ``-4``, etc.) or real values (``1.5m``, ``-6m``, etc.).
+
+  .. image:: _static/height_units.png
+
+  .. image:: _static/height_real_values.png
+
+* **Date Format**
+
+  Used to select the date format. Can be one of four values:
+
+  * *Day/Month/Year*
+  * *Month/Day/Year*
+  * *Year/Month/Day*
+  * *Year/Day/Month*
+
+Audio
+-----
+
+* **Audio Device Dropdown**
+
+  The first dropdown on the Audio tab is a selector for the audio device that OpenRCT2 will use.
+
+* **Master volume**
+
+  A slider used to adjust the volume of OpenRCT2.
+
+* **Sound effects**
+
+  A slider used to adjust the volume of sound effects such as opening/closing menus, guest chatter, guest purchases, etc.
+
+* **Ride music**
+
+  A slider used to adjust the volume of ride music.
+
+* **Disable audio on focus loss**
+
+  Enables/disables muting of audio when OpenRCT2 is not the focused window.
+
+* **Title screen music**
+
+  Selects the music that will be used at the main menu for OpenRCT2. Can be one of four values:
+
+  * *None*
+  * *RollerCoaster Tycoon 1* - requires RCT1 data to work.
+  * *RollerCoaster Tycoon 2* - requires RCT2 data to work.
+  * *Random title music* - randomly select any of the options above.
+
+Controls and Interface
+----------------------
+
+* **Scroll view when pointer at screen edge**
+
+  When enabled, the view of the park will scroll when the mouse cursor is near the edge of the screen.
+
+* **Trap mouse cursor in window**
+
+  When enabled, the mouse cursor will stay locked inside the window.
+
+* **Invert right mouse dragging**
+
+  When enabled, the behavior when panning the view with the right mouse button will be flipped.
+
+* **Zoom to cursor position**
+
+  When enabled, zooming will center on the cursor's location instead of the middle of the screen.
+
+* **Shortcut keys...**
+
+  A button leading to the keyboard shortcuts menu. This window allows modification of the default keyboard shortcuts, and also allows some other functions to have shortcuts assigned.
+
+  To assign a new keyboard shortcut, click the function, then press the key to assign the shortcut. This key can also have modifiers (Shift, Ctrl, Alt).
+
+* **Current Theme**
+
+  A dropdown allowing selection of the theme for the UI. Themes affect all colors of the windows in the game. By default, there are two options, one which matches RCT1, and another which matches RCT2.
```

Comment:
There should be another section about creating custom themes, though that sounds like it should be it's own page.

Eventually this text should link to that.